python/core/symbology-ng/qgsgraduatedsymbolrendererv2.sip

```diff
@@ -108,7 +108,7 @@ class QgsGraduatedSymbolRendererV2 : QgsFeatureRendererV2
 
     virtual QgsGraduatedSymbolRendererV2* clone() const /Factory/;
 
-    virtual void toSld( QDomDocument& doc, QDomElement &element ) const;
+    virtual void toSld( QDomDocument& doc, QDomElement &element, QgsStringMap props ) const;
```

Comment:
What is the best solution here? Add a default value to props? 
```
virtual void toSld( QDomDocument& doc, QDomElement &element, QgsStringMap props = QStringMap( ) ) const;
 virtual void toSld( QDomDocument& doc, QDomElement &element ) const;
```

Or we need to leave the old signature in the bindings? (I hoped python could use the new signature since the additional parameter is optional)